app/controllers/apps_controller.rb

```diff
@@ -25,6 +25,10 @@ def show
   def destroy
     @app = App.find(params[:id])
 
+    @app.services.each do |service|
+      RemoveServiceJob.perform_later(server, @app.clean_name, service)
```

Comment:
Can it cause any trouble if for some reason the `RemoveAppJob` is executed earlier then the `RemoveServiceJob`?